index.html

```diff
@@ -57,7 +57,7 @@
 
 <body>
   <section id='abstract'>
-    <p>This document defines a mechanism enabling web sites to declare a reporting policy that can be used by the user agent to report encountered network errors that prevented it from successfully loading the requested resource, the format of the error reports, and their transmission mechanism.</p>
+    <p>This document defines a mechanism that enables developers to declare a reporting policy for a web application. A user agent can use this policy to report encountered network errors that prevented it from successfully fetching requested resources. Additionally, developers can control the format of the error reports and their transmission mechanism.</p>

```

Comment:
- The first sentence is too broad, it needs to define what said reporting policy is.
- Developers *can't* control the format, that's defined by the spec.

Here's another run: 

> This document defines a mechanism that enables developers to declare a network error reporting policy for a web application. A user agent can use this policy to report encountered network errors that prevented it from successfully fetching requested resources. 

The last bit is probably unnecessary. WDYT?